assets-public/controllers/busy_submit_controller.js

```diff
@@ -0,0 +1,50 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
+  connect () {
+    this.element.addEventListener('turbo:submit-start', (event) => this.disableSubmit(event))
+    this.element.addEventListener('turbo:submit-end', (event) => this.enableSubmit(event))
+  }
+
+  disconnect () {
+    this.enableSubmits()
+  }
+
+  disableSubmit (event) {
+    const button = event.detail.formSubmission.submitter
+    if (button) {
```

Comment:
Niet zeker, maar zou je hier niet checken of het een <button> is? Want in theorie kan je ook een <input type="submit"> hebben, maar daar gaat die spinner niet mee werken.